src/commands/createTransactionCreateMultisignatureAccount.js

```diff
@@ -34,7 +34,7 @@ const description = `Creates a transaction which will register a multisignature
 
 const processInputs = (lifetime, minimum, keysgroup) =>
 	({ passphrase, secondPassphrase }) =>
-		transactions.createMultisignature(
+		transactions.registerMultisignature(
 			passphrase,
 			secondPassphrase,
 			keysgroup,
```

Comment:
In lisk-js 1.0 you would create a transaction using an object with keys and values.

Reply:
Also applies to all the other transactions

Reply:
Ah good catch, thanks.